app/views/how-tos/make-first-prototype/open-prototype-in-editor.html

```diff
@@ -0,0 +1,22 @@
+
+{% set pageTitle = "Open your prototype in your editor" %}
+{% set next = { "title" : "Create pages" , "link" : "create-pages" } %}
+{% set prev = { "title" : "Get started" , "link" : "start" } %}
+
+
+{% extends 'how-tos/make-first-prototype/_BASE.njk' %}
+
+{% block makePrototype %}
+
+<h2>Overview of folders in the Prototype Kit </h2>
+<ul>
+    <li><code>/app</code> is for your work. Inside that folder:</li>
```

Comment:
This first bullet is the lead-in I think?

```suggestion
<p><code>/app</code> is for your work. Inside that folder:</li></p>

<ul>
```